requirements/base.in

```diff
@@ -2,3 +2,8 @@
 
 Django                    # Web application framework
 django-model-utils        # Provides TimeStampedModel abstract base class
+django-autoslug           # Provides AutoSlugField
+unicodecsv                # Allows exporting CSV with unicode support (a drop-in replacement for built-in csv module)
+Pillow                    # Image manipulation module, required to use ImageField
```

Comment:
@mulby Probably not - it was used for `logo` field, which was removed when I was actively trying to wrap this up for the day. Thank you for catching this.